changelog_entry.yaml

```
@@ -0,0 +1,4 @@
+- bump: minor
```

Since it doesn't add new functionality to the package
```suggestion
- bump: patch
```